tests/unit/IonDecimalTest.js

```diff
@@ -175,6 +175,14 @@ define([
             'intValue(-1.0000001)': () => () => assert.equal(ion.Decimal.parse('-1.0000001').intValue(), -1),
         });
 
+        /**
+         * @param decimalString The text representation of the decimal to test.
+         * @param expectedCoefficient The expected `number` or `string` (LongInt) for the coefficient.
+         * @param expectedExponent The expected `number` or `string` (LongInt) for the exponent part.
```

Comment:
always a `number`;  i expect `string` to fail on `util._sign(expectedExponent)`

Reply:
Oh interesting--I noted that it all plumbs down to `LongInt`, which is why I wrote it that way.